### DISCUSSION TITLE
Invert (change) side on the cutting map

### DISCUSSION THREAD

**Discussion by moskitall** [lairdubois/lairdubois-opencutlist-sketchup-extension#640] [Ideas]
![Скриншот 1](https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/assets/168673678/ad44ff54-cc5a-4d0c-8877-82c78b59fa68)
![Скриншот 2](https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/assets/168673678/4306153f-8f6a-4fc9-95b6-08711e04d5bc)
Please add the ability to change the sides of the placement of the part and the order of placement on the cutting map

**Comment by bbeaulant** [comment]
Hi,

We need more detail and explanations to understand your request ...
